### ISSUE TITLE
Bug: Speaking text containing an apostrophe when using SpeakSsml causes a mispronunciation

### ISSUE THREAD

**Issue by m1maker** [nvaccess/nvda#17389]
### Steps to reproduce:
Call nvdaController_speakSsml with this string
"<speak>Bob&apos;</speak>"

### Actual behavior:

NVDA will speak "Bob  '  s"

### Expected behavior:

NVDA should be say "Bob's"

### NVDA logs, crash dumps and other attachments:

Unknown

#### NVDA version:

2024.4

#### Windows version:

Windows 11 24H2

**Comment by Adriani90** [comment]
Which synth are you using? Does this happen when you change it e.g. from onecore to eSpeak?

**Comment by m1maker** [comment]
I am using ESpeak and when I switching to another sinthesizer, this problem is also exists.

**Comment by seanbudd** [comment]
cc @LeonarddeR
